llvm/test/CodeGen/X86/atomic-load-store.ll

```diff
@@ -145,3 +145,40 @@ define <1 x i64> @atomic_vec1_i64_align(ptr %x) nounwind {
   %ret = load atomic <1 x i64>, ptr %x acquire, align 8
   ret <1 x i64> %ret
 }
+
+define <1 x half> @atomic_vec1_half(ptr %x) {
```

Comment:
Add the bfloat test

Reply:
`1 x bfloat` is added here: https://github.com/llvm/llvm-project/pull/120385/files.

Reply:
I'm not sure why bfloat is special such that it needs to be split out